table/block_based/block_based_table_builder.h

```diff
@@ -136,6 +138,24 @@ class BlockBasedTableBuilder : public TableBuilder {
   // Some compression libraries fail when the raw size is bigger than int. If
   // uncompressed size is bigger than kCompressionSizeLimit, don't compress it
   const uint64_t kCompressionSizeLimit = std::numeric_limits<int>::max();
+
+  // Get blocks from mem-table walking thread, compress them and
+  // pass them to the write thread. Used in parallel compression mode only
+  void WriteBlocks(CompressionContext& compression_ctx,
+                   UncompressionContext* verify_ctx);
+
+  // Given raw block content, try to compress it and return result and
+  // compression type
+  void CompressAndVerifyBlock(const Slice& raw_block_contents,
+                              bool is_data_block,
+                              CompressionContext& compression_ctx,
+                              UncompressionContext* verify_ctx,
+                              std::string& compressed_output,
+                              Slice& result_block_contents,
+                              CompressionType& result_compression_type);
+
+  // Get compressed blocks from WriteBlocks and write them into SST
+  void WriteRawBlocks();
```

Comment:
Similar to WriteBlocks(). If my understand is correct, this is the function used by the block writing thread. Can you try to think about a better name? Something like including "thread" in the function name or something like that.

Reply:
Rename `WriteRawBlocks ` to `BGWorkWriteRawBlock`. Get rid of "s" suffix to keep aligned with original `WriteRawBlock`.